integration/graalvm/build.gradle.kts

```diff
@@ -1,5 +1,10 @@
 import java.util.Properties
 
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: remove once KTIJ-19369 / Gradle#22797 is fixed
+plugins {
+    alias(libs.plugins.graalvm.native) apply false
```

Comment:
GraalVM native plugin dynamically generates some "helper" methods so we need to ensure they are generated just once for whole composite project, otherwise build blows up with classpath issues.